### PR TITLE
Lift off rabbitmq

### PR DESCRIPTION
**Pull request by arbulu89**
# Description

Implement basic amqp/rabbitmq configuration.
In order to not break the current runner usage, I have included a new configuration file (`wanda.exs`) so it is used to configure wanda specific processes (rabbitmq in this case). To use it `MIX_ENV=wanda iex -S mix phx.server`.

Besides that, basic boilerplate code.

## Did you add the right label?

Remember to add the right labels to this PR.

## How was this tested?

Some integration test to check if the messages are read properly included as skeleton.

### CODE REVIEW

File: test/trento/application/integration/checks/wanda/messaging/amqp/consumer_test.exs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+      expect(GenRMQ.Processor.Mock, :process, fn %{payload: payload} ->
+        assert ^message = payload
+        send(pid, :consumed)
+        :ok
+      end)
+
+      assert :ok = Publisher.publish_message(message, "results")
+
+      assert_receive :consumed, 1_000
```
Comment (fabriziosestito) [inline test/trento/application/integration/checks/wanda/messaging/amqp/consumer_test.exs:25]: nice! I like the Processor behaviour

**Review by fabriziosestito — APPROVED** [review]
LGTM

**Review by nelsonkopliku — APPROVED** [review]
Cool, thanks!

**Review by dottorblaster — APPROVED** [review]
LGTM

**Pull request merged**